tools/builder/sass.js

```diff
@@ -39,7 +40,7 @@ gulp.task( 'sass:calypsoify', function( done ) {
 		.src( './modules/calypsoify/*.scss' )
 		.pipe( sass( { outputStyle: 'compressed' } ).on( 'error', sass.logError ) )
 		.pipe( banner( '/* Do not modify this file directly.  It is compiled SASS code. */\n' ) )
-		.pipe( autoprefixer() )
+		.pipe( postcss() )
```

Comment:
Clarification/confirmation:

It looks like `autoprefixer` is replaced by `postcss`, but `postcss` uses the config added by this PR which is the config from `calypso-build`:

https://github.com/Automattic/wp-calypso/blob/fecfa1d72f1e244bbd9844488fb900581f1e274c/packages/calypso-build/postcss.config.js#L1-L9

And _that_ config includes autoprefixer, so we still get it. Is that correct?

Seems there are many other gulp tasks using autoprefixer directly, so we still need to depend on the package directly.